packages/payment/src/models/payment-session.ts

```diff
@@ -33,8 +33,8 @@ export default class PaymentSession {
   @Property({ columnType: "text" })
   provider_id: string
 
-  @Property({ columnType: "jsonb", nullable: true })
-  data: Record<string, unknown> | null = null
+  @Property({ columnType: "jsonb" })
+  data: Record<string, unknown> = {}
```

Comment:
q: what pushes you to do that?

Reply:
this is just for consistency since we expect the provider to return a record from `createSession` so this shouldn't be `null`